README.md

```diff
@@ -130,6 +130,18 @@ class E2eTest extends PantherTestCase
 }
 ```
 
+### REPL Debugger
+
+Panther provides a REPL (Read-Eval-Print Loop) for helping you to debug your end-to-end tests thanks to the `bin/panther` binary.
+The Panther client is accessible through a `$p` variable. For instance, try to type something like:
```

Comment:
Maybe should we mention to run `composer require psy/psysh` before running the script?